src/docs/product/profiling/index.mdx

```diff
@@ -61,9 +61,24 @@ The "Slowest Functions" widget shows you a list of the functions that took the m
+Selecting one of the transactions will take you to the **Profile Summary** page below. On this page, you can examine the aggregated information of this profile.
 
-![Transaction Profile summary page](transaction-profile-view.png)
+![Transaction Profile summary page](new-transaction-profile-view.png)
+
+Similar to how previous page displayed information of all of the profiles in your application, the profile summary page displays aggregated information from profiles collected under a single type of transaction.
+
+Using the information on this page, you can dive deeper into what your code was executing during this transaction. The top area of the page includes a percentile chart with profile durations and a search bar. The chart is there so that you can visually identify possible performance regressions or changes - it supports selecting of ranges through a click-drag interaction. Use it to isolate certain date time ranges you would like to investigate. While selecting on date ranges is useful, it is important to be able to narrow down on the population as well, this is why the search bar filters are for. If for example you would like to narrow your investigation to profiles executed under a certain environment, apply the filter and the data will reflect that. Using both of these mechanisms allows you to narrow down on the time range and the population which might be exhibiting performance problems.
+
+Our bottom right side of the page has aggregated information about function regressions that Sentry may have detected during the selected time range as well as a list of the slowest functions in your application as well as some general information about the profile. We can see information like when was this profile last seen, p75, p95 and p99 values as well as the total amount of profiles we collected for this transaction.
+
+Below the general information, we have a list of regressions that Sentry may have detected as well as a list of the slowest functions executed during this transactions.
+Clicking on the before/after timings in the regressed function list will take you to example flamecharts of the before and after execution so that you can investigate what changed.
+
+The slowest functions list serves as an overview of functions which are consuming the most time in your application - these are usually good candidates for optimization.
+
+The left side of the page contains am aggregate flamegraph as well as digest of the slowest and most regressed functions. We recommend reading the docs about [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) so you can understand what this data is telling us. In short, looking for wide frames that have no children is a good first thing to look at - this will allow you to identify frames that are often occupying the stack and are good candidates for optimization.
```

Comment:
```suggestion
There's an aggregate flamegraph as well as a digest of the slowest and most regressed functions on the left side of the page. As a rule of thumb, a good first thing to look for is wide frames that have no children. This will allow you to identify frames that are often occupying the stack and may be good candidates for optimization. 

We recommend reading the [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) docs to get a better understand of this data.


```